content/docs/components/serving/tfbatchpredict.md

```diff
@@ -1,9 +1,15 @@
 +++
 title = "TensorFlow Batch Predict"
-description = "Batch prediction for TensorFlow models"
+description = "See Kubeflow [v0.6 docs](https://v0-6.kubeflow.org/docs/components/serving/tfbatchpredict/) for Batch Prediction with TensorFlow models"
 weight = 60
 +++
 
+[Batch Predict](https://github.com/kubeflow/batch-predict) is not supported in Kubeflow versions greater than 0.6.
+
+{{% alert title="Alpha version" color="warning" %}}
```

Comment:
Thanks for composing this alpha alert! But for now, let's follow the same pattern that we followed for the Chainer docs. So, remove all content from the page except for line 7 above which states that the feature is not supported above Kubeflow v0.6.

Reply:
Done!